features/checkout/addressing_order/order_address_validation.feature

```diff
@@ -16,12 +16,20 @@ Feature: Order address validation
         When the visitor specify the email as "jon.snow@example.com"
         And the visitor try to specify the incorrect billing address as "Ankh Morpork", "Frost Alley", "90210", "United Russia" for "Jon Snow"
         And the visitor complete the addressing step
-        Then I should be notified that "United Russia" country does not exist
+        Then they should be notified that "United Russia" country does not exist
```

Comment:
Why `they`? When the context changed from singular to plural?

Reply:
It's [singular they](https://en.wikipedia.org/wiki/Singular_they) 💃